<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeforeAfterHistogramDialog</class>
 <widget class="QDialog" name="BeforeAfterHistogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="HistogramWidget" name="beforeHistogramWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>581</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>570</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Before</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>After</string>
   </property>
  </widget>
  <widget class="HistogramWidget" name="afterHistogramWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>581</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="beforeImageLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="afterImageLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>301</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistogramWidget</class>
   <extends>QWidget</extends>
   <header>HistogramWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
